syntax.qmd

```diff
@@ -175,7 +175,7 @@ There are a few exceptions, which should never be surrounded by spaces:
     )
     ```
 
-    Note that single-sided formulas with a complex right-hand side do need a space:
+    Note that single-sided formulas with a complex right-hand side do need a space.
```

Comment:
I am fine with `:` here and above, could you explain more?

Reply:
I just thought it would be in the spirit of things to make it uniform for all consecutive bullet points

Reply:
Nope, just better to be consistent.